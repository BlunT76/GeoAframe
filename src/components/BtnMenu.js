import React from 'react';
import PropTypes from 'prop-types';

const BtnMenu = (props) => {
  const { handleOpenMenu } = props;
  return (
    <button type="button" onClick={() => handleOpenMenu()} />
  );
};

BtnMenu.propTypes = {
  handleOpenMenu: PropTypes.func,
};

BtnMenu.defaultProps = {
  handleOpenMenu: () => {},
};

export default BtnMenu;
