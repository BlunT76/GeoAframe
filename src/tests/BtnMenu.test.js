import React from 'react';
import renderer from 'react-test-renderer';
import BtnMenu from '../components/BtnMenu';

test('BtnMenu is always the same', () => {
  const component = renderer.create(
    <BtnMenu />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
