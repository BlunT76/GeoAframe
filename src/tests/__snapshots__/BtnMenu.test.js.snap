// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BtnMenu is always the same 1`] = `
<button
  className="btn btn-light BtnMenu"
  onClick={[Function]}
  type="button"
>
  <svg>
    bars-solid.svg
  </svg>
</button>
`;
